.github/workflows/e2e.yml

```
@@ -31,6 +31,12 @@ jobs:
             echo "is_pr=false" >> "$GITHUB_OUTPUT"
           fi
 
+      - name: Add comment to PR
+        id: pr_comment
+        if: steps.check_pr.outputs.is_pr == 'true'
+        run: |
+          gh pr comment "${{ github.event.inputs.pr_or_branch }}" -b "e2e workflow launched on this PR: [View run](https://github.com/${repository.owner}/${repository.repo}/actions/runs/${process.env.GITHUB_RUN_ID})"
```

```suggestion
          gh pr comment "${{ github.event.inputs.pr_or_branch }}" -b "e2e workflow launched on this PR: [View run](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})"
```

This change got lost somehow.

---

sigh, sorry ... i broke it with a rebase and forced push I'm sure ...